config/manager/manager.yaml

```diff
@@ -56,5 +56,16 @@ spec:
           requests:
             cpu: 10m
             memory: 64Mi
+        env:
+          - name: AWS_ACCESS_KEY_ID
```

Comment:
I didn't try this but I just thought that `AWS_CREDENTIAL_PROFILES_FILE` can be used to avoid the secret values in the environment variables. CCO is supposed to create the credentials file as a field in the secret.

Reply:
Apparently the environment variable is `AWS_SHARED_CREDENTIALS_FILE`
https://github.com/aws/aws-sdk-go/issues/3514#issuecomment-808556848